charts/prometheus/templates/sts.yaml

```
@@ -318,9 +318,13 @@ spec:
 {{ toYaml .Values.server.extraVolumes | indent 8}}
 {{- end }}
 {{- if .Values.server.persistentVolume.enabled }}
+  {{- $volumeName := "storage-volume" }}
+  {{- if ne .Values.server.persistentVolume.statefulSetNameOverride "" }}
+  {{- $volumeName = .Values.server.persistentVolume.statefulSetNameOverride }}
+  {{- end }}
```

Why not use `default`? Or you might consider merge with line 323 / 327 below
```suggestion
  {{- $volumeName := default "storage-volume" .Values.server.persistentVolume.statefulSetNameOverride "" }}
```